Bulldozer.CSV/Maps/FinancialTransaction.cs

```diff
@@ -0,0 +1,295 @@
+                    {
+                        var newFinancialTransactionImport = new FinancialTransactionImport()
+                        {
+                            FinancialTransactionForeignKey = $"{ImportInstanceFKPrefix}^{financialTransactionCsv.Id}",
+                            AuthorizedPersonForeignKey = $"{ImportInstanceFKPrefix}^{financialTransactionCsv.AuthorizedPersonId}",
+                            BatchForeignKey = $"{ImportInstanceFKPrefix}^{financialTransactionCsv.BatchId}",
+                            Summary = financialTransactionCsv.Summary,
+                            TransactionCode = financialTransactionCsv.TransactionCode,
+                            TransactionDate = financialTransactionCsv.TransactionDate.ToSQLSafeDate(),
+                            CurrencyTypeValueId = this.CurrencyTypeValues[financialTransactionCsv.CurrencyType].Id,
+                            TransactionTypeValueId = this.TransactionTypeValues[financialTransactionCsv.TransactionType.ToString()].Id,
+                            CreatedByPersonForeignKey = $"{ImportInstanceFKPrefix}^{financialTransactionCsv.CreatedByPersonId}",
+                            CreatedDateTime = financialTransactionCsv.CreatedDateTime.ToSQLSafeDate(),
+                            ModifiedByPersonForeignKey = $"{ImportInstanceFKPrefix}^{financialTransactionCsv.ModifiedByPersonId}",
+                            ModifiedDateTime = financialTransactionCsv.ModifiedDateTime.ToSQLSafeDate(),
+                            FinancialTransactionDetailImports = new List<FinancialTransactionDetailImport>(),
+                        };
+
+                        if ( financialTransactionCsv.TransactionSource.IsNotNullOrWhiteSpace() )
+                        {
+                            newFinancialTransactionImport.TransactionSourceValueId = this.TransactionSourceTypeValues[financialTransactionCsv.TransactionSource].Id;
+                        }
+                        else
+                        {
+                            // set default source to onsite, exceptions listed below
+                            newFinancialTransactionImport.TransactionSourceValueId = this.TransactionSourceTypeValues["On-Site"].Id;
+                        }
+
+                        var nonCashAssetDV = this.NonCashAssetTypeValues.GetValueOrNull( financialTransactionCsv.NonCashAssetType );
+                        if ( nonCashAssetDV != null )
+                        {
+                            newFinancialTransactionImport.NonCashAssetValueId = nonCashAssetDV.Id;
+                        }
+
+                        var creditCardTypeDV = this.CreditCardTypeValues.GetValueOrNull( financialTransactionCsv.CreditCardType );
+                        if ( creditCardTypeDV != null )
+                        {
+                            newFinancialTransactionImport.CreditCardTypeValueId = creditCardTypeDV.Id;
+                        }
```

Comment:
Support adding defined values

Reply:
This is already handled in CSVComponent.csv before hand: https://github.com/KingdomFirst/Bulldozer/pull/84/files#diff-414b7cfa5b49e0802e816844a0729c2b0b7357578a8fcc0071731b93f96ba4daR478